Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Transformer.purs

```diff
@@ -68,8 +70,18 @@ getLocation predcition = {
   , alpha : 1.0
   , fullAddress : dummyAddress
   , locationItemType : Just PREDICTION
+  , distance : getFormattedDistance (fromMaybe 0 (prediction ^._distance))
+  , showDistance : checkShowDistance (fromMaybe 0 (prediction ^._distance))
 }
 
+getFormattedDistance :: Int -> String
+getFormattedDistance distanceInMeters
+  | distanceInMeters >= 1000 = parseFloat (toNumber distanceInMeters / 1000.0) 1 <> " km"
```

Comment:
@prashant601  Don't repeat same functions. Instead define it in one file and import it in other. Create only one function for getFormattedDistance and checkShowDistance.